civictechprojects/static/css/partials/_AboutProject.scss

```diff
@@ -227,10 +227,17 @@
     width: 100%;
   }
 }
-
-.AboutProjects-recent-activity {
-  padding: 20px; /*as with description-details consider maxwidth */
-
+/* events section overrides from Event page style to About Project page style */
+.AboutProjects-events {
```

Comment:
It's a bit awkward that `<EventCardsListings />` uses an H1 here (it's fine on the actual Events page) so we'll probably want to fix this for proper hierarchy purposes later, but I made it display as other section headings do.